spec/controllers/books_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+RSpec.describe BooksController do
+  describe '#index' do
+    let!(:books) { create_list(:book, 3) }
+
+    before { get :index }
+
+    it 'returns http success' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders index template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'assingns @books' do
+      expect(assigns(:books)).to eq(books.sort_by(&:title))
```

Comment:
Use `match_array` instead of `eq`
```suggestion
      expect(assigns(:books)).to eq(books)
```